Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -297,10 +297,13 @@ bool HasMovedOrRotated()
             bool rotated = lastRotation != targetComponent.transform.rotation;
 
             // save last for next frame to compare
-            lastPosition = targetComponent.transform.position;
-            lastRotation = targetComponent.transform.rotation;
-
-            return moved || rotated;
+            bool change = moved || rotated;
```

Comment:
You should check if the position or rotation changed by some threshold.  As is,  even just floating point rounding errors could trigger a change detection

Reply:
Unlikely, the previous code did the same direct comparison and it had trouble detecting small changes. I also threw some debug statements and saw that the compare was different only when rotating. Therefore i believe the threshold check is built in. But if you want an adjustable threshold...

```suggestion
           // moved since last time we checked it?
        bool HasMovedOrRotated()
        {
            // moved or rotated?
            bool moved = lastPosition != targetComponent.transform.position;
            //bool rotated = lastRotation != targetComponent.transform.rotation;
            bool rotated = Quaternion.Angle(lastRotation, targetComponent.transform.rotation) > 0.1f;

            // save last for next frame to compare
            bool change = moved || rotated;
            if (change)
            {
                lastPosition = targetComponent.transform.position;
                lastRotation = targetComponent.transform.rotation;
            }
            return change;
        }
```

Reply:
Unlikely, the original did the same check and didn't detect small rotations; also i tossed som debug statements is and saw no activity when things were not moving. However this might simply be because i am only rotating around the Y axis. Anyway here it is...

`// moved since last time we checked it?
        bool HasMovedOrRotated()
        {
            // moved or rotated?
            bool moved = lastPosition != targetComponent.transform.position;
            //bool rotated = lastRotation != targetComponent.transform.rotation;
            bool rotated = Quaternion.Angle(lastRotation, targetComponent.transform.rotation) > 0.1f;

            // save last for next frame to compare
            bool change = moved || rotated;
            if (change)
            {
                lastPosition = targetComponent.transform.position;
                lastRotation = targetComponent.transform.rotation;
            }
            return change;
        }`